### ISSUE TITLE
"runtime" parameter breaks training / resampling

### ISSUE THREAD

**Issue by a-hanf** [a-hanf/mlr3automl#43]
Something broke:
```
model = AutoML(tsk("iris"), runtime=60)
Warning message:
The fallback learner 'response' and the base learner 'prob' have different predict types 
model$train()
Error in makeActiveBinding(name, active[[name]], public_bind_env) : 
  symbol already has a regular binding
Called from: makeActiveBinding(name, active[[name]], public_bind_env)
```

It works without the runtime parameter

**Comment by a-hanf** [comment]
Found the offending commit, working on fix:
https://github.com/mlr-org/mlr3/commit/063fb94272ec58c64d85732fcdf2476d0c5c629e

**Comment by a-hanf** [comment]
Fixed in #44 
If you encounter this issue, please try updating mlr3 to the most recent Github version
